suites/nautilus/interop/test_ceph_sanity_rpm.yaml

```diff
@@ -0,0 +1,88 @@
+#===============================================================================================
```

Comment:
Likewise the above comments apply to this file also... that is

- rename the file to `test-ceph-rpms.yaml`
- check the test configurations used for deployment
- check the ansi_config